### PR TITLE
Fix description for tidb_opt_mpp_outer_join_fixed_build_side

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
Fix description for tidb_opt_mpp_outer_join_fixed_build_side

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -1861,7 +1861,7 @@ mysql> desc select count(distinct a) from test.t;
 - Persists to cluster: Yes
 - Type: Boolean
 - Default value: `ON`
-- Specifies whether to allow using outer table as the build side in outer join for TiFlash.
+- When the variable value is `ON`, left join operations always use inner tables as the build side and right join operations always use outer tables as the build side. If you set the value to `OFF`, outer join operations can use either side of the tables as the build side.
```
Comment (fixdb) [inline system-variables.md:1864]: When the variable is set to `ON`, left join operator always uses inner table as the build side and right join operator always uses outer table as the build side.

Comment (TomShawn) [reply system-variables.md:1864]:
```suggestion
- When the variable value is `ON`, the left join operator always uses inner table as the build side and the right join operator always uses outer table as the build side. If you set the value to `OFF`, outer join operations can use either side of the tables as the build side.
```

File: system-variables.md [hunk 2]
```diff
@@ -1861,7 +1861,7 @@ mysql> desc select count(distinct a) from test.t;
 - Persists to cluster: Yes
 - Type: Boolean
 - Default value: `ON`
-- Specifies whether to allow using outer table as the build side in outer join for TiFlash.
+- When the variable value is `ON`, the left join operator always uses inner table as the build side and the right join operator always uses outer table as the build side. If you set the value to `OFF`, outer join operations can use either side of the tables as the build side.
```
Comment (fixdb) [inline system-variables.md:1864]: operations -> operator

Comment (fixdb) [reply system-variables.md:1864]: If the value is set to `OFF`, outer join operator can use either side of the tables as the build side.

Comment (TomShawn) [reply system-variables.md:1864]:
```suggestion
- When the variable value is `ON`, the left join operator always uses inner table as the build side and the right join operator always uses outer table as the build side. If you set the value to `OFF`, the outer join operator can use either side of the tables as the build side.
```

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Pull request merged**